style.css

```diff
@@ -75,6 +81,110 @@ body {
   color: #3a4a42;
 }
 
+.popup_container {
+  width: 100%;
+  height: 100%;
+  background: white;
+  padding: 1rem;
+  position: fixed;
+  display: none;
+  top: 0;
+  left: 0;
+  z-index: 20;
+  overflow: scroll;
+}
+
+.popup_mobile {
+  display: inline-block;
+  width: 90%;
+}
+
+.popup_desktop {
+  display: none;
+}
+
```

Comment:
@tresorsawasawa 
Sorry, about the mobile pop up close button, it's an image and if I should place it on the main image it won't be obvious to see that is why I placed it outside, the figma design is displaying white and it's not white when I downloaded it.

About the fonts, color and others, I do copy those ones directly from figma design so I don't know why anyone is different there 

About the desktop image, the screen size you reference in this review is much larger than the screen size in figma design. 

And please I don't understand the blur effect you want me to add cause I can't notice any blur effect in the figma design. Thanks

Reply:
Hi @Fayob 

- For the close button I can suggest you use the same design as on the Desktop. You can put it at the top-right where it will be vertically aligned at the same line with its below content(project image,...). In short, it should have the same right spacing as other content,  so vertically aligned at the same line,

- I think you should first resize your Popup to have some spacing around it, so you can see the background. Right now the background is not visible because your popup container fills the entire screen. And I think even the 1st reviewer mentioned about that point(for the **width** of the popup)

- Once you are able to see the background you can learn about **background blur** in **[this article](https://www.w3schools.com/howto/howto_css_blurred_background.asp)**